lms/static/js/dashboard/views/course_view.js

```diff
@@ -0,0 +1,233 @@
+                        ),
+                        course_name: _.sprintf(
+                            '<span id="unenroll_course_name">%(courseName)s</span>', {courseName: courseName}
+                        ),
+                        cert_name_long: _.sprintf(
+                            '<span id="unenroll_cert_name">%(certNameLang)s</span>', {certNameLang: certNameLang}
+                        )
+                    }, true));
+
+                    $('#refund-info').html(refundInfo);
+                    $('#unenroll_course_id').val(courseId);
+                },
+
+                emailSettings: function (e) {
+                    var $currentTarget = this.$(e.currentTarget);
+
+                    $('#email_settings_course_id').val($currentTarget.data('course-id'));
+                    $('#email_settings_course_number').text($currentTarget.data('course-number'));
+
+                    if ($currentTarget.data('optout') === 'False') {
+                        $('#receive_emails').prop('checked', true);
+                    }
+                },
+
+                upgradeToVerified: function (e) {
+                    var $currentTarget = this.$(e.currentTarget);
+                    $currentTarget.closest('.action-upgrade').data('user');
+                    $currentTarget.closest('.action-upgrade').data('course-id');
+                },
+
+                unRegisterBlockCourse: function (e) {
+                    var $currentTarget = this.$(e.currentTarget),
+                        courseId = $currentTarget.data('course-id'),
+                        courseNumber = $currentTarget.data('course-number'),
+                        courseName = $currentTarget.data('course-name');
+
+                    if (this.$('#block-course-msg').length) {
+                        $('.disable-look-unregister').click();
+                    }
+
+                    $('#track-info').html(_.sprintf(
+                        '<span id="unenroll_course_number">%(courseNumber)s</span> ' +
+                        '- <span id="unenroll_course_name">%(courseName)s?</span>',
+                        {courseNumber: courseNumber, courseName: courseName})
+                    );
+
+                    $('#unenroll_course_id').val(courseId);
+                },
+
+                unEnrollFormSubmitted: function (e, xhr) {
+                    if (xhr.status === 200) {
+                        location.href = this.settings.dashboard;
+                    } else if (xhr.status === 403) {
+                        location.href = this.settings.signin_user + "?course_id=" +
+                            encodeURIComponent(this.$("#unenroll_course_id").val()) + "&enrollment_action=unenroll";
+                    } else {
+                        $('#unenroll_error').html(
+                            xhr.responseText ? xhr.responseText : gettext("An error occurred. Please try again later.")
+                        ).stop().css("display", "block");
```

Comment:
CSS classes should be added/removed instead of using JavaScript to update CSS values.